### DISCUSSION TITLE
Supporting NoSQL data sources ?

### DISCUSSION THREAD

**Discussion by corridordigital** [amundsen-io/amundsen#1651] [Ideas]
Hello ! 
Google Firestore, MongoDB, Cassandra, etc.
Would that be such a bad idea ? does it make sense to index metadatao for data sources that don't enforce a schema ? 
I'm asking because Elastic is supported. Why not other sources ?

**Comment by feng-tao** [comment]
we do have an extractor for cassandra (https://github.com/amundsen-io/amundsen/blob/main/databuilder/databuilder/extractor/cassandra_extractor.py)

**Comment by mgorsk1** [comment]
We are definitely not against it and would be happy to review any PR submitted with such extractors ;)
